randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -540,10 +570,11 @@ Extra - asset_id: 14
       Morph Ball
   > Door to Terminator Room
       Any of the following:
-          Screw Attack or Speed Booster
+          Screw Attack
           All of the following:
               Morph Ball
               Morph Ball Bombs or Power Bombs
+          Speed Booster and Shinespark (Beginner)
```

Comment:
Can you explain how to do the shinespark? looking at the room, im confused on how that should be done i

Reply:
Its a 2-tap shortcharge from the Alcatraz escape, updated with shortcharge and changed template to CanLayAnyBombs